src/components/WebLogView/Row.tsx

```diff
@@ -8,24 +8,44 @@ import { TableCellWithTooltip } from '../TableCellWithTooltip'
 import { HighlightedText } from '../HighlightedText'
 import { getRequestType } from './WebLogView.utils'
 import { SearchResults } from './SearchResults'
+import { useMemo } from 'react'
 
 interface RowProps {
   data: ProxyDataWithMatches
   isSelected?: boolean
   onSelectRequest: (data: ProxyDataWithMatches) => void
   filter?: string
+  highlightedRequestIds?: string[]
 }
 
-export function Row({ data, onSelectRequest, isSelected, filter }: RowProps) {
+export function Row({
+  data,
+  onSelectRequest,
+  isSelected,
+  filter,
+  highlightedRequestIds,
+}: RowProps) {
+  const opacity = useMemo(() => {
+    if (!highlightedRequestIds) {
+      return 1
+    }
+
+    return highlightedRequestIds.includes(data.id) ? 1 : 0.4
+  }, [highlightedRequestIds, data.id])
```

Comment:
I saw @Llandy3d's comment about having more noticeable difference, but I'd like to challenge the idea of using opacity here mostly for accessibility/usability reasons:
For example, when I select a rule, I may want to edit its filter based on the request list. Having low opacity makes this pretty hard. Do you have alternative ideas?

Reply:
Colors could be a good approach also to differentiate between requests matched for the extraction and requests matched for the replacing 🤔 
I think this is a good followup item we've been discussing but for a different PR

Reply:
As discussed, I'll work on improved highlighting and present in a separate PR